engine/execution/storehouse.go

```diff
@@ -80,6 +88,9 @@ type InMemoryRegisterStore interface {
 
 type OnDiskRegisterStore = storage.RegisterIndex
 
+// pebble.Registers is an implementation of OnDiskRegisterStore interface
+var _ OnDiskRegisterStore = (*pebble.Registers)(nil)
```

Comment:
Unrelated to this PR, but a similar change as the FinalizedReader interface check above. It ensures the implementation, and also acts as a shortcut to find the implementation.